InnoClinic.OfficesAPI/DAL/Entities/Office.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MongoDB.Bson;
+
+namespace DAL.Entities;
+
+public sealed class Office
+{
+    public ObjectId Id { get; set; }    
+    public byte[]? Photo { get; set; }  
+    public string City { get; set; }
+    public string Street { get; set; }
+    public string HouseNumber { get; set; }
+    public string OfficeNumber { get; set; }
+    public string RegistryPhoneNumber { get; set; } 
+    public bool Status { get; set; }
+    public string Address { get; set; }
+
+    public Office(string city, string street, string houseNumber, string officeNumber, string registryPhoneNumber, bool status)
```

Comment:
Usually we don't use a constructor in entities, to make correct `Address` you can do something like that with property:
`public string Address => $"{City}, {Street}, {HouseNumber}, {OfficeNumber}";`

Reply:
I would like to have Address in the DTO. Why you need to store duplicated filed?
There is no address filed in your doc too

Reply:
Fare enough